### ISSUE TITLE
Comment is taked into account for closing brace oO ?

### ISSUE THREAD

**Issue by Poil** [rodjek/vim-puppet#24]
I have a code like this (this one must not be valid)
When running auto indent It give me that

``` ruby
class toto {
  case toto {
    titi: {
    }
    # toto
    default: {
    }
    # toto
    }
    # toto
    }
```

If I comment in indent/puppet.vim

``` vim
"    if pline =~ '^\s*#'
"        return ind
"    endif
```

Auto indent is working correctly

``` ruby
class toto {    
  case toto {
    titi: {     
    }
    # toto      
    default: {
    }
    # toto  
  }
 # toto     
}
```

**Comment by lelutin** [comment]
indentation works great with the use case presented in the top of the issue without that if block. I'd suggest just dropping it

**Comment by lelutin** [comment]
this was fixed in #100
